src/main/reactjs/src/interfaces/authorisation.ts

```diff
@@ -1,9 +1,4 @@
-import { ClerkLanguagePair } from 'interfaces/language';
-
-export interface LanguagePair {
-  from: string;
-  to: string;
-}
+import { ClerkLanguagePair, PublicLanguagePair } from 'interfaces/language';
```

Comment:
Tarvitaanko enää ClerkLanguagePairia?

Reply:
Se `permissionToPublish` tulee vielä `ClerkLanguagePair`in kautta.

Reply:
Aivan